AK/Math.h

```diff
@@ -540,7 +524,7 @@ template<FloatingPoint T>
 constexpr T pow(T x, T y)
 {
     CONSTEXPR_STATE(pow, x, y);
-    // fixme I am naive
+    // Fixme: I am naive
```

Comment:
Some editors only recognize `FIXME`, and we usually use that in all the other places as well (even in this PR).